framework/src/Volo.Abp.AspNetCore.Mvc/Volo/Abp/AspNetCore/Mvc/AbpAspNetCoreMvcModule.cs

```diff
@@ -104,10 +106,29 @@ public override void ConfigureServices(ServiceConfigurationContext context)
             }
         });
 
+        var abpAspNetCoreMvcOptions = context.Services.ExecutePreConfiguredActions<AbpAspNetCoreMvcOptions>();
+
         var mvcCoreBuilder = context.Services.AddMvcCore(options =>
         {
             options.Filters.Add(new AbpAutoValidateAntiforgeryTokenAttribute());
+
+            options.CacheProfiles.Add(AbpApplicationLocalizationScriptController.GetCacheProfileName, new CacheProfile
+            {
+                Duration = abpAspNetCoreMvcOptions.ScriptCacheDuration,
+                VaryByQueryKeys = new []{ "hash" }
```

Comment:
use `_v` instead of `hash` in querystring (everywhere). Hashing is our internal way to implement the versioning, and it is an implementation detail (I specifically indicated that in [my comment](https://github.com/abpframework/abp/issues/16075#issuecomment-1486740668)).